source/mainnet/net/guides/become-baker.rst

```diff
@@ -309,20 +315,13 @@ the ``baker add`` command as shown in the following:
 Finalization
 ------------
 
-Finalization is the voting process performed by nodes in the *finalization
-committee* that *finalizes* a block when a sufficiently big number of members of the committee have received the block and agree on its outcome. Newer blocks
-must have the finalized block as an ancestor to ensure the integrity of the
-chain. For more information about this process, see :ref:`finalization<glossary-finalization>`.
+Finalization is the process by which a block is marked to be “finalized”, i.e., part of the authoritative chain. Transactions that are part of finalized blocks are considered authoritative. New blocks can be only added following the last finalized block to ensure the integrity of the chain. The finalization process is conducted by the bakers with an effective stake amount of at least 0.1% of the total amount (effective stake) of CCD in pools, known as the Finalization committee. Total stake in pools is referred to as total stake in pools without Passive Delegation.
```

Comment:
I think it's fine. But maybe we should revise this paragraph again when the actual finalization committee parameters has been chosen for mainnet.